src/com/PacketSerializer.cpp

```diff
@@ -681,4 +680,26 @@ namespace plug::com
         header1.setDSP(DSP::none);
         return {{Packet<EmptyPayload>{header0, EmptyPayload{}}, Packet<EmptyPayload>{header1, EmptyPayload{}}}};
     }
+
+    std::array<Packet<EmptyPayload>, 3> serializeInitCommand_V3_USB()
+    {
```

Comment:
Packet data in this function is copied from the attached JSON capture file, at lines 4631, 4867, 5103.

As yet I have no understanding of the meaning of these packets.